practice-02/js/index.js

```diff
@@ -72,3 +85,24 @@ document.addEventListener('DOMContentLoaded', function () {
         });
     });
 });
+
+// Handle click event on logo buttons
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
We've already have "DOMContentLoaded" event listener

Reply:
Yes, I combined them for the same event

Reply:
Yes, I fixed